app/services/association_authorizer.rb

```diff
@@ -0,0 +1,27 @@
+class AssociationAuthorizer
+  def self.authorized_associations(associations,  klass_name = nil, shouldRaiseException = true)
```

Comment:
Please use underscores in variable names and not camel case for consistency.  Ditto in the method below.